packages/ozone/src/tag-service/index.ts

```diff
@@ -22,15 +22,18 @@ export class TagService {
     ]
   }
 
-  async evaluateForSubject() {
+  // Allow the caller to seed the initial tags
+  async evaluateForSubject(initialTags: string[] = []) {
     try {
-      const tags: string[] = []
+      const tags = new Set([...initialTags])
```

Comment:
You don't need to copy here (the `Set` will do that for you). This also allows to type the argument in a broader way `initialTags?: Iterable<string>` instead of defaulting to an empty array.

```suggestion
      const tags = new Set(initialTags)
```

Reply:
cleaned it up as recommended!